app/Http/Livewire/Album.php

```diff
@@ -42,7 +42,7 @@ public function mount(AlbumModel $album, AlbumFactory $albumFactory)
 	 */
 	public function render()
 	{
-		switch (Configs::getValueAsString('layout')) {
+		switch (Configs::getValueAsString('layout', '1')) {
```

Comment:
Why does this one get a default?

Reply:
Must have been an oversight. My IDE was also complaining that the second parameters was unused. There were some more of those error. I fixed them in https://github.com/LycheeOrg/Lychee/pull/1336/commits/fa2d3a3840a50565e70a3430be8d31c49729399c